projects/jdwp/codegen/new_type_generator.py

```diff
@@ -0,0 +1,21 @@
+"""New type Generator."""
+from projects.jdwp.defs.schema import PrimitiveType
+
+
+def get_python_type(jdwp_type: str) -> str:
+    """Map JDWP type to Python type."""
+    mapping = {
```

Comment:
If `jdwp_type` was of type `PrimitiveType` then we will be forced to update this mapping when we make incompatible changes to the `PrimitiveType` enum. In particular, at some point we should remove `dict` - it is not a valid type. It would be great if type checker/a runtime test could also detect that this mapping needs to be updated.